packages/message-relayer/bin/run.ts

```diff
@@ -1,109 +0,0 @@
-import { Wallet, providers } from 'ethers'
-import { Bcfg } from '@eth-optimism/core-utils'
-import { Logger, LoggerOptions } from '@eth-optimism/common-ts'
-import * as Sentry from '@sentry/node'
-import * as dotenv from 'dotenv'
-import Config from 'bcfg'
-
-import { MessageRelayerService } from '../src'
-
-dotenv.config()
-
-const main = async () => {
```

Comment:
We can entirely cut out this configuration and execution file because the new base service handles all of this for us.